### DISCUSSION TITLE
(Not really a bug) Snackbar when a page has been saved successfully is very annoying

### DISCUSSION THREAD

**Discussion by chenaoxd** [requarks/wiki#3592] [Error / Bug Report]
**Describe the bug**
Snackbar when a page has been saved successfully is very annoying, and you can't close it. Especially when used in mobile, it covers the close button so you have to wait for the snackbar to disappear.

**To Reproduce**
1. Just edit a page and click the save button

**Expected behavior**
At least provide a close button or make clicking the other area can close the snackbar.

**Screenshots**
![image](https://user-images.githubusercontent.com/3839678/88533028-25413d80-d038-11ea-8884-d29a904e8abd.png)

**Host Info (please complete the following information):**
 - OS: Ubuntu 16.04, Docker
 - Wiki.js version: 2.4.107
 - Database engine: sqlite

**Additional context**
No

**Comment by 3DSkaoi** [comment]
So, here's my question for you. How could you confirm it was successfully saved if you have no confirmation message?

If ~5 seconds is too much for you, you can press the back button on your mobile device.

**Comment by netrc** [comment]
I agree it's annoying. Saving pages to a wiki is a pretty common occurrence; Confirmations should be unobtrusive and non-modal. Just a small 'Saved!' text in a bottom of screen status line would be enough. Neither Wikipedia nor Confluence use any sort of pop-up saved notification.
